app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @comments = Comment.all.paginate page: params[:page]
```

Comment:
For this specifically I don't really care since the paginate call will always be the last call in the chain.